fpga/include/villas/fpga/card/platform_card.hpp

```diff
@@ -0,0 +1,35 @@
+/** 
+ * Author: Pascal Henry Bauer <pascal.bauer@rwth-aachen.de>
```

Comment:
Please add your self, and also me and Daniel via separate `SPDX-FileCopyrightText:` lines as below.